seismicpro/src/refractor_velocity/refractor_velocity.py

```diff
@@ -357,29 +364,14 @@ def _get_constraints(self):
             "fun": lambda x: x[:1][self._empty_layers[:1]] - np.array([self.init['t0']])[self._empty_layers[:1]]}
         return [constraint_offset, constraint_velocity, constraint_freeze_velocity, constraint_freeze_t0]
 
-    def _fit_regressor(self, x, y, start_slope, start_time):
-        """Method fits the linear regression by given data and initial values.
-
-        Parameters
-        ----------
-        x : 1d ndarray of shape (n_samples, 1)
-            Training data.
-        y : 1d ndarray of shape (n_samples,)
-            Target values.
-        start_slope : float
-            Starting coefficient to fit a linear regression.
-        start_time : float
-            Starting intercept to fit a linear regression.
-
-        Returns
-        -------
-        params : tuple
-            Linear regression `coef` and `intercept`.
-        """
-        lin_reg = SGDRegressor(loss='huber', penalty=None, shuffle=True, epsilon=0.05, eta0=0.1, alpha=0.01, tol=1e-6,
-                               max_iter=1000, learning_rate='optimal')
-        lin_reg.fit(x, y, coef_init=start_slope, intercept_init=start_time)
-        return lin_reg.coef_[0], lin_reg.intercept_
+    @staticmethod
+    def _scale_standart(data):
+        """Scale data to zero mean and unit variance."""
+        if len(data) == 0:
+            return data, np.nan, np.nan
+        mean, std = np.mean(data), np.std(data)
+        data_scaled = (data - mean) / (std + 1e-10)
```

Comment:
I'd suggest using `normalization.scale_standard`:

```suggestion
        data_scaled = normalization.scale_standard(data, mean, std, 1e-10)
```

Reply:
Also move normalization from `gather.utils` to `utils`